PyInstaller/lib/modulegraph/modulegraph.py

```diff
@@ -35,9 +37,14 @@
 from . import zipio
 from ._compat import get_instructions, BytesIO, StringIO, \
      pathname2url, _cOrd, _READ_MODE
+from .node import Node, Alias, AliasNode, BadModule, \
+     ExcludedModule, MissingModule, Script, BaseModule, \
+     BuiltinModule, SourceModule, InvalidSourceModule, \
+     CompiledModule, InvalidCompiledModule, Extension, \
+     Package, NamespacePackage, RuntimeModule, \
+     RuntimePackage, FlatPackage, ArchiveModule
```

Comment:
Just to preempt complaints, using the `*` is not recommended because it leads to confusion about where the imports are from.